### PR TITLE
Chinese (Simplified) translations

### PR DESCRIPTION
**Pull request by axamdy**
Hello.
These files should be saved as UTF-8 without BOM format, otherwise they will can not be displayed correctly in-game.

### CODE REVIEW

File: locale/shine/extensions/ban/zhCN.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"REASON": "理由：",
+	"Bans": "封禁",
+	"BANNED_BY": "操作人",
+	"EXPIRY": "期限",
+	"NEVER": "永不",
+	"DATE_FORMAT": "%d %B %Y %H:%M",
+	"UNBAN": "解封",
+	"LOAD_MORE": "载入更多",
+	"LOAD_MORE_TIP": "从服务器载入更多数据。",
+	"PLAYER_BANNED": "已封禁{TargetName} {Duration:NonZero:for }{Duration:Duration:permanently} ({Reason}).",
```
Comment (Person8880) [inline locale/shine/extensions/ban/zhCN.json]: The `{Duration:NonZero:for }` bit here will be English `for ` when `Duration` isn't 0 here. You just need to replace the `for ` bit at the end.

Similarly for `{Duration:Duration:permanently}`, when the duration is 0 it'll display `permanently`.

File: locale/shine/extensions/commbans/zhCN.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+	"Comm Bans": "指挥员黑名单",
+	"PLAYER_BANNED": "从指挥员名单里拉黑了{TargetName} {Duration:NonZero:for }{Duration:Duration:permanently}。",
+	"BANNED_WARNING": "你已被指挥员名单拉黑{Duration:NonZero:for }{Duration:Duration:permanently}，无法指挥。",
```
Comment (Person8880) [inline locale/shine/extensions/commbans/zhCN.json]: These two have the same issue as for the bans translation.

**Comment by Person8880** [comment]
Thanks for the translations. There's a few small bits missing that I've commented on, after that I can merge.

File: locale/shine/extensions/ban/zhCN.json (hunk 1 above)
Comment (axamdy) [inline locale/shine/extensions/ban/zhCN.json]: Oh okay. Now I understand that meaning, have updated.

**Comment by axamdy** [comment]
Thanks for review. Now those stuffs has been fixed and I've re-updated the files.

**Pull request merged**